src/components/company/CompanyFacts.astro

```diff
@@ -90,10 +90,13 @@ const reportURL =
             <div class="flex flex-wrap items-center justify-between gap-x-4 gap-y-1 text-sm">
               <h3>Omsättning ({year})</h3>
               <p class="text-sm text-orange-250">
-                {company.baseFacts[year].turnover.toLocaleString('sv-SE', {
-                  maximumFractionDigits: 0,
-                }) +
-                  ' ' +
+                {(company.baseFacts[year].turnover / 1000_000).toLocaleString(
+                  'sv-SE',
+                  {
+                    maximumFractionDigits: 0,
+                  },
+                ) +
+                  ' m' +
```

Comment:
```suggestion
                  ' M' +
```

Good catch! Didn't think about this since the unit is SEK and not MSEK.

Let's use capital M for million.

Reply:
Of course :)